test/commandLine.test.js

```diff
@@ -34,30 +35,30 @@ describe('CommandLineManager', function () {
       argv = argv.concat(['--proxy-host', 'host', '--proxy-port', '9687']);
       CommandLineManager.processArgs(argv);
       console.log.restore();
-      expect(RdGlobalConfig.proxy.host).to.eql('host');
+      expect(RdGlobalConfig.proxy.host).to.eql(proxy_host_actual_value);
       expect(RdGlobalConfig.proxy.port).to.eql(9687);
     });
 
     it('remove any protocol part from proxy-host', function () {
```

Comment:
are these tests valid now? are we removing any protocol part from the proxy host that we provide as the input?

Reply:
not addressed yet.

Reply:
and below as well...

Reply:
Fixed

Reply:
can be resolved.